core/src/mindustry/input/Placement.java

```diff
@@ -54,6 +55,32 @@
         return points;
     }
 
+    public static Seq<Point2> upgradeConveyorLine(int startX, int startY, int endX, int endY){
```

Comment:
These two functions are nearly the same, and use conduit/conveyor-specific implementation details. This isn't acceptable.

Reply:
Yeah but Conduit and Conveyor is not the same thing. And also there won't be any new such blocks.